slsReceiverSoftware/src/slsReceiverTCPIPInterface.cpp

```diff
@@ -111,7 +111,7 @@ void slsReceiverTCPIPInterface::startTCPServer() {
     while (true) {
         try {
             auto socket = server->accept();
-			socket.setReceiveTimeout(10000);
+			socket.setReceiveTimeout(static_cast<int>(5E6));
```

Comment:
It's not a good coding style to have constant numbers in the code. Better define it somewhere.
Instead of 5E6 I find 5*1000*1000 more readable. The compiler will optimizes it away.

Reply:
True, fixed it.